src/models/searchservice/response/directanswer/StringValueDirectAnswer.ts

```diff
@@ -0,0 +1,12 @@
+import { EnumOrLiteral } from '../../../utils/EnumOrLiteral';
+import { BaseFieldValueDirectAnswer } from './BaseFieldValueDirectAnswer';
+import { BuiltInFieldType } from './BuiltInFieldType';
+
+/**
+ * A direct answer whose source is a string or string list field in the knowledge graph.
+ *
+ * @public
+ */
+export interface StringValueDirectAnswer extends BaseFieldValueDirectAnswer<string | string[]> {
```

Comment:
I was thinking a number of field types (instagram handle, facebook url, linkedin url, etc.) are better suited for the name StringValueDirectAnswer, since their values can truly only be string and never string[] (afaict anyways).
Is there some other name we could give for this interface? I had a hard time thinking of one besides say `TextValueDirectAnswer` which didn't seem ideal for me.

Reply:
We could also have an interface for those types and lock it down. For example, we'd have:
`FacebookUrlDirectAnswer extends StringValueDirectAnswer<string>`.